vagrant/Vagrantfile

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  config.vm.box = "debian/jessie64"
```

Comment:
Why use jessie when stretch is out for more than a year?

Reply:
this was random, i used an other vagrant file as template, where i used jessie. If you want, I can test it with stretch?

Reply:
done